### ISSUE TITLE
Home page navbar too low

### ISSUE THREAD

**Issue by rachejazz** [smaranjitghose/awesome-portfolio-websites#824]
Anyone else facing this?
Browser: Firefox
![image](https://user-images.githubusercontent.com/42383989/123553582-a0a3f580-d799-11eb-9222-7a97639b721e.png)

**Comment by smaranjitghose** [comment]
Which browser are you using?

**Comment by smaranjitghose** [comment]
> Which browser are you using?

I checked. This issue is not detected on most chromium-based browsers like Edge, Google Chrome, or Brave. However, when we use Firefox this exactly what we get.

**Comment by rachejazz** [comment]
There's a default padding maybe given by Firefox but none by any other Browser. Should I close this or let this be as a bug?

**Comment by deeqakkk** [comment]
> There's a default padding maybe given by Firefox but none by any other Browser. Should I close this or let this be as a bug?

Did you tried fixing it in your local files?and let us know if fixing it for firefox creates any issues in other browsers.

**Comment by smaranjitghose** [comment]
> There's a default padding maybe given by Firefox but none by any other Browser. Should I close this or let this be as a bug?

No this is a serious bug we need to consider

**Comment by rachejazz** [comment]
Sure, I'll sit with this when i get a break again. Please do consider one day skip. Exam day tomorrow :)

**Comment by smaranjitghose** [comment]
Sure @rachejazz .  I can understand. You can take a day or two off for this. All the best for your exam

**Comment by harshit078** [comment]
I will try to fix this bug!!

**Comment by rachejazz** [comment]
@smaranjitghose I saw this only on firefox:
![image](https://user-images.githubusercontent.com/42383989/124016751-9c7b1080-da03-11eb-8a93-9c7ca9be3c0d.png)

but on chrome, there's none:
![image](https://user-images.githubusercontent.com/42383989/124016667-88371380-da03-11eb-8d56-a2f9e12009fe.png)

Hey @harshit078 thanks for taking this up :))
Hope this svg discovery helps you!

**Comment by smaranjitghose** [comment]
@rachejazz Yes, the avg animation is supposed to appear for a sidebar in smaller screens and not in the nav bar for laptop screen sizes

**Comment by rachejazz** [comment]
Cool then @harshit078 can easily now fix some more specific css tags to omit that out

**Comment by smaranjitghose** [comment]
Any updates @harshit078 ?

**Comment by harshit078** [comment]
> Any updates @harshit078 ?

ya sorry kinda missed few previous messages and I have been working on the bug and I will pull the request and upload it within 2 days

**Comment by anushbhatia** [comment]
Any updates @harshit078 ?

**Comment by harshit078** [comment]
I am using safari and there seemed no issue for that and checked for chrome too, I have adjusted the position Margin left and margin right attribute. I am fixing for Firefox too
